### PR TITLE
22313-Packages-are-created-for-extension-methods-that-does-not-are-exactly-the-package-name

### PR DESCRIPTION
**Pull request by demarey**
Just fixing names from https://github.com/pharo-project/pharo/pull/1685

### CODE REVIEW

**Comment by Ducasse** [comment]
I do not get why certain trait methods are failing on windows????
I relaunched the ci.

**Comment by MarcusDenker** [comment]
er have random crashes, both on win and macOS, they are not related to the issues

File: src/Monticello-Tests/MCWorkingCopyForExtensionsTest.class.st [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Class {
+	#name : #MCWorkingCopyForExtensionsTest,
+	#superclass : #MCTestCase,
+	#category : #'Monticello-Tests-Base'
+}
+
+{ #category : #tests }
+MCWorkingCopyForExtensionsTest >> tearDown [
```
Comment (jecisc) [inline src/Monticello-Tests/MCWorkingCopyForExtensionsTest.class.st:8]: This should be in "running" protocol

File: src/Monticello-Tests/MCWorkingCopyForExtensionsTest.class.st [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	super tearDown.
```
Comment (jecisc) [inline src/Monticello-Tests/MCWorkingCopyForExtensionsTest.class.st]: The super tearDown should be the last call of the tearDown

**Review by jecisc — CHANGES_REQUESTED** [review]

File: src/Monticello-Tests/MCWorkingCopyForExtensionsTest.class.st (hunk 2 above)
Comment (astares) [reply src/Monticello-Tests/MCWorkingCopyForExtensionsTest.class.st]: Yes - there even is a lint rule for this

**Comment by estebanlm** [comment]
was this addressed @demarey ?

**Comment by demarey** [comment]
@estebanlm Just now

**Comment by astares** [comment]
I'm confused ... @demarey / @estebanlm what is the status/necessary to move this forward?

Should we just restart the CI because the failing zinc test seem to be unrelated

**Comment by demarey** [comment]
Sorry for the delay, I just saw this email because of a problem with my email filters.
I just triggered a new ci build for this PR.
In the same time, I do not know if this PR is still mergeable. Is there a way to ask GitHub to check again?

**Comment by jecisc** [comment]
I think github check for mergeability each time you load the PR page or each time you act on the PR.

**Comment by jecisc** [comment]
Tests failure are not related and will happen if development is not merged into the branch. (It's the date tests)

**Comment by astares** [comment]
Checks have passed - ready to integrate?

**Pull request merged**